jvm/src/main/kotlin/app/cash/trifle/VerifiedData.kt

```diff
@@ -0,0 +1,40 @@
+package app.cash.trifle
+
+import app.cash.trifle.internal.TrifleAlgorithmIdentifier
+import app.cash.trifle.internal.providers.JCAContentVerifierProvider
+import app.cash.trifle.internal.validators.CertChainValidatorFactory
+import okio.ByteString.Companion.toByteString
+import app.cash.trifle.protos.api.alpha.Certificate as CertificateProto
```

Comment:
do you need these aliases?

Reply:
removed.  A little annoyed that they didn't auto-remove with my formatting but.... maybe I can fix that later.